example/package.json

```diff
@@ -1,30 +1,21 @@
 {
-  "name": "example",
-  "version": "1.15.0",
-  "private": true,
-  "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
-    "start": "react-native start",
-    "test": "jest",
-    "lint": "eslint ."
-  },
-  "dependencies": {
-    "react": "16.9.0",
-    "react-native": "0.68.5",
-    "@getyoti/react-native-yoti-doc-scan": "file:.."
-  },
-  "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@babel/runtime": "^7.8.4",
-    "@react-native-community/eslint-config": "^0.0.7",
-    "babel-jest": "^25.1.0",
-    "eslint": "^6.8.0",
-    "jest": "^25.1.0",
-    "metro-react-native-babel-preset": "^0.59.0",
-    "react-test-renderer": "16.9.0"
-  },
-  "jest": {
-    "preset": "react-native"
-  }
+    "name": "example",
+    "main": "index.js",
+    "private": true,
+    "scripts": {
+      "android": "react-native run-android",
+      "ios": "react-native run-ios",
+      "start": "react-native start"
+    },
+    "dependencies": {
+      "react": "^17.0.2",
```

Comment:
I'm not a big fan of version ranges, since they might introduce changes we might not want/anticipate. 🤔

Reply:
the range includes minors + patches. not majors, so it's fine